src/pages/home/report/ContextMenu/ContextMenuActions.js

```diff
@@ -325,7 +325,7 @@ export default [
             !isArchivedRoom &&
             !isChronosReport &&
             !ReportUtils.isConciergeChatReport(reportID) &&
-            reportAction.actorEmail !== CONST.EMAIL.CONCIERGE,
+            reportAction.actorAccountID !== CONST.ACCOUNT_ID.CONCIERGE,
```

Comment:
Oh hi! We can flag Concierge messages now so I don't think this is working as expected.

Reply:
Hmm in dev or stag / prod too?? :O

Reply:
Just curious why it won't work, as in this line we simply changed from email to accountID

Reply:
I wonderrrr if this is a dev thing - maybe @dangrous have you set `EXPENSIFY_ACCOUNT_ID_CONCIERGE` to your concierge account id in your dev env?

Reply:
ooh, no! didn't see all those new env variables. Will add them now. NEVER MIND!

Reply:
yep that worked, sorry for the false alarm!

Reply:
Fantastic!! I'm gunna go clean up the pee in my pants, back in a jiffy 👀